jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java

```diff
@@ -0,0 +1,143 @@
+/********************************************************************************
+ * Copyright (c) 1998, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1998, 2018 IBM Corporation. All rights reserved.
```

Comment:
In new files copyright year should be 2018 only and copyright holder should the one submitting the PR - so in this case you. In the other words - first line should be removed and second line should not contain 1998.

Reply:
@lukasj